bower.json

```diff
@@ -45,7 +45,8 @@
     "px-theme": "^3.0.0",
```

Comment:
Bump dependency for px-app-helpers to `^2.1.0` to get the new released with the favoritable changes

Reply:
:+1: